### PR TITLE
Update translations for 0.38

### LINKED ISSUE #14071: Pull in translations for new strings for 0.38 release
Here's how things are looking currently: 11 of 21 are back to 100%. When we get close to the final release, we'll need to:
-[x] Get unfinished translations to 100%
-[x] Pull in the translations

image

### PR DESCRIPTION
**Pull request by mazameli**
Closes #14071

### CODE REVIEW

**Comment by mazameli** [comment]
Not sure why all these tests are failing. I merged in master 'cause I'd branched this off a few weeks ago in order to update POEditor with the new 0.38 strings and wanted to make sure I could merge back in cleanly, but maybe I did something wrong?

**Comment by paulrosenzweig** [comment]
@mazameli there are conflict markers in `src/metabase/models/dashboard.clj` that break everything. Since your changes don't affect that file, you could check it out from master and commit it.

File: frontend/src/metabase/visualizations/lib/chart_values.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import _ from "underscore";
 
+import { t } from "ttag";
```
Comment (paulrosenzweig) [inline frontend/src/metabase/visualizations/lib/chart_values.js:3]: This is probably from a merge conflict also?

File: frontend/test/metabase/internal/__snapshots__/components.unit.spec.js.snap [hunk 2]
```diff
@@ -1150,11 +1150,11 @@ exports[`Select should render "kitchen_sink" correctly 1`] = `
     >
       <span>
         Entity Key
-        , 
+        ,
```
Comment (paulrosenzweig) [inline frontend/test/metabase/internal/__snapshots__/components.unit.spec.js.snap:1153]: Same with this whitespace change?

**Review by paulrosenzweig — APPROVED** [review]
Two other files have unrelated changes, but looks good otherwise!

**Pull request merged**